### PR TITLE
Bump src-cli min version to 3.32.0 and re-gen docs

### PR DESCRIPTION
**Pull request by mrnugget**
Release chores.

### CODE REVIEW

File: doc/cli/references/batch/preview.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 | `-force-override-ignore` | Do not ignore repositories that have a .batchignore file. | `false` |
 | `-get-curl` | Print the curl command for executing this query and exit (WARNING: includes printing your access token!) | `false` |
 | `-insecure-skip-verify` | Skip validation of TLS certificates against trusted chains | `false` |
-| `-j` | The maximum number of parallel jobs. Default is GOMAXPROCS. | `16` |
+| `-j` | The maximum number of parallel jobs. Default is GOMAXPROCS. | `8` |
```
Comment (eseliger) [inline doc/cli/references/batch/preview.md:18]: not sure about this but honestly it doesn't matter much

**Review by eseliger — APPROVED** [review]

File: doc/cli/references/batch/preview.md (hunk 1 above)
Comment (mrnugget) [reply doc/cli/references/batch/preview.md:18]: Hey, I tried my best and actually set `GOMAXPROCS=8` when generating these. That lead to _other_ places not being updated, but these two.

Comment (eseliger) [reply doc/cli/references/batch/preview.md:18]: haha alright alright 😁

Comment (mrnugget) [reply doc/cli/references/batch/preview.md:18]: Okay, pushed a commit that always sets `GOMAXPROCS=8` when generating the docs. That way it's stable in the future.

**Pull request merged**